src/client/components/stockDataRegion/index.test.js

```diff
@@ -0,0 +1,80 @@
+import { expect } from 'chai';
+
+import { mapStateToProps } from '.';
+import * as selectors from '../../selectors';
+
+describe('mapStateToProps', () => {
+  const mockState = {
```

Comment:
Just define the state within each spec.  You only use this particular state in one spec anyways.  Also this state is very simple so imo there's no benefit in overwriting 1 or 2 properties in a spec; rewriting it from scratch is the same or fewer lines of code and makes what's happening more explicit.